java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -0,0 +1,145 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.ZaddOptions;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Sorted set Commands interface for both standalone and cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=sorted-set">Sorted set Commands</a>
+ */
+public interface SortedSetBaseCommands {
+
+    /**
+     * Adds members with their scores to the sorted set stored at <code>key</code>.<br>
+     * If a member is already a part of the sorted set, its score is updated.
+     *
+     * @see <a href="https://redis.io/commands/zadd/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param membersScoresMap A <code>Map</code> of members to their corresponding scores.
+     * @param options The Zadd options.
+     * @param changed Modify the return value from the number of new elements added, to the total
+     *     number of elements changed.
+     * @return The number of elements added to the sorted set. <br>
+     *     If <code>changed</code> is set, returns the number of elements updated in the sorted set.
+     *     <br>
```

Comment:
do you need linebreak at the end of the block?
```suggestion
     *
```